src/contexts/AzureMapDataSourceContext.tsx

```diff
@@ -31,9 +33,7 @@ const AzureMapDataSourceStatefulProvider = ({
   dataFromUrl,
   collection
 }: IAzureDataSourceStatefulProviderProps) => {
-  const [dataSourceRef] = useState<atlas.source.DataSource>(
-    new atlas.source.DataSource(id, options)
-  )
+  const [dataSourceRef] = useState<atlas.source.DataSource>(new source.DataSource(id, options))
```

Comment:
useState<source.DataSource>

Reply:
types from 'atlas' don't work in that way 😔